src/game/Unit.cpp

```diff
@@ -2099,69 +2099,91 @@ MeleeHitOutcome Unit::RollMeleeOutcomeAgainst(const Unit* pVictim, WeaponAttackT
 
     // bonus from skills is 0.04%
     int32 skillBonus  = 4 * (attackerWeaponSkill - victimMaxSkillValueForLevel);
-    int32 sum = 0;
-    int32 roll = urand(0, 10000);
-    int32 tmp = miss_chance;
+    int32 roll = urand(0, 100) * 100;
```

Comment:
using this formula will give only random number 100 by 100 are you sure about that change?